lib/Swagger/src/TemplateClassification/Storage/Api/TCSApi.php

```diff
@@ -274,90 +268,80 @@ public function getTemplateTypeWithHttpInfo($wiki_id, $page_id)
         $resourcePath = str_replace("{format}", "json", $resourcePath);
 
         
-        
-  
         // for model (json/xml)
         if (isset($_tempBody)) {
             $httpBody = $_tempBody; // $_tempBody is the method argument, if present
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)